### PR TITLE
Added validation and unit tests to InflowBoardController ES

### PR DESCRIPTION
**Pull request by ellieSlann**
still need to add happy path tests

### CODE REVIEW

File: InflowAutomationTests/InflowBoardControllerTests.cs [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+    [TestFixture]
+    public class InflowBoardControllerTests
+    {
+        public MockFileSystem _mockFileSystem;
+
+        [SetUp]
+        public void setup()
+        {
+            var Boris = new Dictionary<string, MockFileData> {
+                {@"\Dev\Boris.json", new MockFileData("{}") },
```
Comment (JDavidMartin) [inline InflowAutomationTests/InflowBoardControllerTests.cs]: Boris.Json made me laugh

**Comment by JDavidMartin** [comment]
Other than no happy path test I'm fine with it.
I'll leave that up to you if you want to add it in before a merge, or just merge

File: InflowAutomationTests/InflowBoardControllerTests.cs (hunk 1 above)
Comment (ellieSlann) [reply InflowAutomationTests/InflowBoardControllerTests.cs]: lol I'd forgotten I'd called it that! I should probably change it to something more descriptive...

**Comment by ellieSlann** [comment]
I'm not going to merge yet, two of the tests are not passing because IDirectory is not implemented correctly and I need to work out why.

**Pull request merged**